support-frontend/assets/helpers/tracking/acquisitions.js

```diff
@@ -283,16 +274,54 @@ function deserialiseReferralData(serialised: string): Object {
   };
 }
 
+
+// Reads the acquisition data from sessionStorage.
+function getReferrerAcquisitionDataFromSessionStorage(): ?ReferrerAcquisitionData {
```

Comment:
This is a copy-paste and rename of the deleted `readReferrerAcquisitionData` above.